templates/email/default/problem-confirm-not-sending.html

```diff
@@ -22,7 +22,7 @@ <h1 style="[% h1_style %]">Please confirm your&nbsp;report</h1>
   <p style="[% p_style %]">If you no longer wish to publish this report, please take no further action.</p>
   [% end_padded_box %]
 </th>
-[% WRAPPER '_email_sidebar.html' object = report %]
+[% WRAPPER '_email_sidebar.html' object = report, url = token_url %]
```

Comment:
Yeah, in some places we use commas, in others we use just spaces, and in others we use linebreaks. Shall I standardise on spaces then?